test/test_connectionpool.py

```diff
@@ -275,11 +275,6 @@ def test_contextmanager(self):
         self.assertRaises(ClosedPoolError, pool._get_conn)
         self.assertRaises(Empty, old_pool_queue.get, block=False)
 
-    def test_mixed_case_url(self):
-        pool = HTTPConnectionPool('Example.com')
-        response = pool.request('GET', "http://Example.com")
```

Comment:
So what's not necessarily entirely clear is that this is a follow-on PR to #1054, where I suggested that rather than dirty up all the tests that currently use the network we should move to make them either use localhost (if possible), or have a single decorator that we can use to skip tests that need the network when the network is unavailable.